src/wazuh_modules/inventory_sync/src/agentSession.hpp

```diff
@@ -111,13 +111,15 @@ class AgentSessionImpl final
         auto globalVersion = data->global_version();
 
         auto agentIdString = std::string(agentId.data(), agentId.size());
+
```

Comment:
Does modifying this file make sense?

Reply:
These changes were added by astyle. The checks validate for all files, not only the one that you changed. So, I have to push this change.